crypto/ocsp/ocsp_integration_test.cc

```diff
@@ -18,6 +19,35 @@
 
 std::string GetTestData(const char *path);
 
+// Certificates always expire and that applies to the root certificates we're
+// using in our trust store too.
+static void isCertificateExpiring(X509 *cert) {
+  // Get the current time and add 1 year to it.
+  int64_t warning_time = time(nullptr) + 31536000;
+
+  if (X509_cmp_time_posix(X509_get_notAfter(cert), warning_time) < 0) {
+    fprintf(stdout, "\n\n\n\n");
+    X509_NAME_print_ex_fp(stdout, X509_get_subject_name(cert), 0, 0);
+    fprintf(stdout, " WILL EXPIRE IN A YEAR, PLEASE REPLACE ME WITH THE NEW"
+            " EXPECTED ROOT CERTIFICATE FROM THE ENDPOINT.\n\n\n\n");
+  }
+}
```

Comment:
10/10. Did you test this locally by changing the warning time to 15 years and see it print out?

Hypothetically could we add a cloudwatch alarm on a regex looking at the codebuild logs to ticket us?

Reply:
Yeah I tested it by adding more time to it and got warnings.

The test output only seems to get dumped out when a test failure happens, but I think we can configure an alarm to warn us. Test failures always happen anyways :)

Reply:
I created `CryptoAlg-1894` to track making the alarm. I looked into it briefly, but I'm not well-versed enough in CloudWatch and ticketing configurations to make this a one-day task.